causeway-client/pom.xml

```diff
@@ -65,9 +65,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.keycloak</groupId>
-      <artifactId>keycloak-core</artifactId>
-      <scope>test</scope>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-annotations</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-jsr310</artifactId>
```

Comment:
How are these 2 deps related to the keycloak version change?

Reply:
I removed keycloak from causeway-client, which was sneaking these dependencies here.